metecho/api/admin.py

```diff
@@ -115,6 +115,25 @@ class ProjectSlugAdmin(admin.ModelAdmin):
 class TaskAdmin(admin.ModelAdmin):
     list_display = ("name", "project", "deleted_at")
     list_filter = (SoftDeletedListFilter,)
+    fields = (
+        ("name", "project"),
+        "description",
+        ("branch_name", "org_config_name"),
+        "commits",
+        "origin_sha",
+        "ms_commits",
+        "has_unmerged_commits",
+        ("pr_number", "pr_is_open"),
+        ("review_submitted_at", "review_valid", "review_status", "review_sha"),
+        ("reviewers", "get_all_users_in_commits"),
+        "status",
+        ("assigned_dev", "assigned_qa"),
+    )
+    readonly_fields = (
+        "commits",
+        "reviewers",
+        "get_all_users_in_commits",
```

Comment:
It's not very pretty, but it's there! 🎉  This is fine for now, until we know how we want to access it.

![Screenshot 2020-07-16 18 37 59](https://user-images.githubusercontent.com/552316/87729553-ab859600-c793-11ea-87c8-7165f0a2b732.png)